urllib3/response.py

```diff
@@ -183,7 +186,7 @@ def _init_decoder(self):
             if content_encoding in self.CONTENT_DECODERS:
                 self._decoder = _get_decoder(content_encoding)
 
-    def _decode(self, data, decode_content, flush_decoder):
+    def _decompress(self, data, decode_content, flush_decoder):
```

Comment:
We do refer to this as decoding everywhere else, I'm -0.5 on renaming it to decompress for consistency.